src/Result.php

```diff
@@ -6,8 +6,10 @@
 
 final class Result
 {
+    public const ERROR_KEY_SEPARATOR = '.';
```

Comment:
Using `.` directly without constant looks way better to me. There's not that many chances to mistype with `.` as with longer strings.

Reply:
If I got this correctly, this was suggested in https://github.com/yiisoft/validator/issues/147#issuecomment-1004499792 by @xepozz. I'm fine with using it without constant too because it's such a common notation in other languages / frameworks.

Reply:
:heavy_check_mark: